### ISSUE TITLE
Feature Request: Heatmap Data like streak

### ISSUE THREAD

**Issue by mmfs37** [Richardsl/heatmap-calendar-obsidian#8]
It would be amazing if it could show below the heatmap some data about that. 
Options could be: 
Current Streak, Longest Streak, Accuracy (acc since jan1 or acc since first entry ever), average amount (probably only possible with numbers and not with words or true and false)

![image](https://user-images.githubusercontent.com/100991030/159954189-9d59a2e1-7f96-42d5-b5a5-ba18ba845714.png)
As design inspirate I would submit the design from the anki heatmap addon. 
Just the clean heatmap and at the bottom the stat and the a amount left to it.

Coding implementation for the dataview could look like: 
Current_Streak: True
Longest_Streak: False
Accurace: Beginning

or anything else

**Comment by Richardsl** [comment]
yes i actually thought of something like that aswell. maybe even with a horizontal slider where you could choose an week range (say 3weeks) and it would automatically calculate the avarage for the range you selected.

I'm currently pretty busy with work, so i dont have a whole lot of time to work on features, but maybe ill get some more time soon. Until then i'm happy to accept pull requests from others
